src/tools/ResultsComparer/Data.cs

```diff
@@ -167,4 +169,4 @@ private static string GetSimplifiedProcessorName(string processorName)
             return processorName.Replace(" ", "");
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Weird that this closing paren shows up. Can we check if the scripts still work to prove that this was unintentional?

Reply:
it's a newline at the end of the file (added)

Reply:
Yeah, I'm not sure why that newline was added. Every time I tried to delete it it comes back. Is it a problem?

Reply:
I don't think it should be a problem. One quick check you could do is rerun the script, however, as per @EgorBo's comment, I think we should be good.